sdk/netapp/Azure.ResourceManager.NetApp/api/Azure.ResourceManager.NetApp.net8.0.cs

```diff
@@ -2687,4 +2767,31 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NetApp.Models.UpdateNetworkSiblingSetContent>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NetApp.Models.UpdateNetworkSiblingSetContent>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    public partial class UsageName : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.NetApp.Models.UsageName>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NetApp.Models.UsageName>
```

Comment:
would it make more sense if this is renamed to `NetAppUsage`?
This looks like to be a complicated construct which is not really a "name"

Reply:
Not sure it does, that might be confused with `UsageResult` of which `UsageName` is a property.  It is only the name value and localized name value.

Reply:
before our renaming, we have `UsageResult` and `UsageName`.
Now that since `UsageResult` -> `NetAppUsageResult`, I think it should be reasonable that we change `UsageName` -> `NetAppUsageName`, right?